bundle/Controller/LocationController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace EzSystems\EzContentOnTheFlyBundle\Controller;
+
+use eZ\Publish\API\Repository\LocationService;
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use eZ\Publish\Core\REST\Server\Controller;
+use eZ\Publish\Core\REST\Server\Values;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+class LocationController extends Controller
+{
+    protected $configResolver;
+    
+    protected $locationService;
+
+    public function __construct(ConfigResolverInterface $configResolver, LocationService $locationService)
+    {
+        $this->configResolver = $configResolver;
+        $this->locationService = $locationService;
+    }
+
+    public function suggested(Request $request, $content)
+    {
+        $response = new JsonResponse();
+        
+        $configuration = $this->configResolver->getParameter('content', 'ez_contentonthefly');
```

Comment:
You should inject the parameter from the service's definition rather than pull it using the config resolver: `$content;ez_contentonthefly$`. Use a setter for this rather than a constructor argument.